### PR TITLE
[aot] Support texture arguments for AOT kernels

### PR DESCRIPTION
**Pull request by jim19930609**
Issue: #

### CODE REVIEW

File: python/taichi/aot/utils.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from taichi.types.ndarray_type import NdarrayType
 from taichi.types.texture_type import TY_CH2FORMAT, RWTextureType, TextureType
 
-template_types = (NdarrayType, TextureType, template)
+template_types = (NdarrayType, RWTextureType, TextureType, template)
```
Comment (ailzhang) [inline python/taichi/aot/utils.py:10]: Is this accidentally included?

**Review by ailzhang — APPROVED** [review]
Thanks!

File: python/taichi/aot/utils.py (hunk 1 above)
Comment (jim19930609) [reply python/taichi/aot/utils.py:10]: Ah this is by mistake, thanks!

**Review by YuCrazing — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**